monitoring.go

```diff
@@ -116,6 +116,7 @@ type SnowflakeQueryStatus struct {
 // SnowflakeConnection is a wrapper to snowflakeConn that exposes API functions
 type SnowflakeConnection interface {
 	GetQueryStatus(ctx context.Context, queryID string) (*SnowflakeQueryStatus, error)
+	QueryContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Rows, error)
```

Comment:
Why is this exposure necessary?

Reply:
Not necessary any longer - we found the following code used for GetStatus and will plan to have clients do this analogously. We'll also add to go.doc a similar example.

```
err = conn.Raw(func(x interface{}) error {
// Execute asynchronous query
rows, _ := x.(driver.QueryerContext).QueryContext(ctx, "select 1", nil)
defer rows.Close()

// Retrieve and check results of the query after casting the result
status := rows.(SnowflakeResult).GetStatus()
```